core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -754,17 +754,32 @@ class AutoTuner(
       limitedLogicList: Option[Seq[String]] = Some(Seq("spark.sql.shuffle.partitions")),
       showOnlyUpdatedProps: Boolean = true):
       (Seq[RecommendedPropertyResult], Seq[RecommendedCommentResult]) = {
-    filterByUpdatedPropertiesEnabled = showOnlyUpdatedProps
-    limitedLogicList.foreach { limitedSeq =>
-      limitedSeq.foreach(_ => limitedLogicRecommendations.add(_))
-    }
-    skipList.foreach(skipSeq => skipSeq.foreach(_ => skippedRecommendations.add(_)))
-    if (processPropsAndCheck) {
-      initRecommendations()
-      calculateRecommendations()
+    val isOnGpu = {
+      val isPluginLoaded = getPropertyValue("spark.plugins") match {
+        case Some(f) => f.equals("com.nvidia.spark.SQLPlugin")
+        case None => false
+      }
+      val rapidsEnabled = getPropertyValue("spark.rapids.sql.enabled") match {
+        case Some(f) => f.toBoolean
+        case None => true
+      }
+      isPluginLoaded && rapidsEnabled
+    }
+    if (!isOnGpu) {
+      appendComment("Only GPU eventLogs are supported")
```

Comment:
The comment "_Only GPU eventLogs are supported_" is not clear enough to the end-user.
I suggest to make it more verbose. i.e., 
```
AutoTuner recommendations only support eventlogs generated by Spark applications running on GPU hardware with Spark RAPIDS Plugin enabled
```
Then we can add which flag is missing

```
XYZ is disabled, XYZ should be enabled
```